commons/phone/phone_bluetooth_devices.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "org.radarcns.phone",
+  "type": "record",
+  "name": "PhoneBluetoothDevices",
+  "doc": "Phone Bluetooth devices info.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "device timestamp in UTC (s)"},
+    {"name": "timeReceived", "type": "double", "doc": "device receiver timestamp in UTC (s)"},
+    {"name": "pairedDevices", "type": ["null", "int"], "doc": "Number of paired devices"},
+    {"name": "nearbyDevices", "type": "int", "doc": "Number of nearby devices"},
```

Comment:
Could you set this type to `["null", "int"]` as well, and I just realised, for both the default can also be set to `"default": null`.

Reply:
Done